### DISCUSSION TITLE
Open source Windows Subsystem for Linux?

### DISCUSSION THREAD

**Discussion by bplasmeijer** [microsoft/WSL#9941] [General]
Hello,

I do ask myself why Windows Subsystem for Linux is not open-sourced.

It's part of Windows, yes, but looking at all issues, 1.9K. It would be great if the community can help.

My 5 cents, like or dislike.

cc: @craigloewen-msft @benhillis

**Comment by benhillis** [comment]
Agreed it would be nice to investigate at some point.
